guides/expo-react-native.md

```diff
@@ -0,0 +1,302 @@
+sidebar_position: 6
+---
+
+import Arcade, {Clip} from '../src/components/Arcade.tsx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import InvocationTip from '../docs/workflows/\_invocation-type-tip.mdx';
+import RNExpoInstallation from '../docs/workflows/\_sdk-setup/\_installation-react-native-expo.mdx';
+
+# Integrate Beyond Identity Passwordless Authentication into Expo
+
+This guide provides information on how to set up Beyond Identity as a passwordless authentication provider for a React Native application that uses Expo.
+
+This guide will cover:
+
+- How to install the React Native SDK
+- How to configure Beyond Identity as an Identity Provider
+- How to create an identity and generate a passkey
+- How to authenticate with a passkey
+
+## Prerequisites
+
+- Set up a [developer account](https://developer.beyondidentity.com/docs/v1/workflows/account-setup)
+
+## Install the React Native SDK
+
+In order to use Beyond Identity functionality in your application, you will need to install the SDK. The [React Native SDK](https://github.com/gobeyondidentity/bi-sdk-react-native) provides functionality from passkey creation to passwordless authentication. A set of functions are provided to you through an `Embedded` namespace.
+
+The React Native SDK is a wrapper around our native SDKs ([Android](https://github.com/gobeyondidentity/bi-sdk-android) and [iOS](https://github.com/gobeyondidentity/bi-sdk-swift)), so it has [custom native code](https://docs.expo.dev/workflow/customizing/). Expo Go only directly works with libraries in the Expo SDK, so to leverage the Beyond Identity React Native SDK and other libraries outside of Expo Go, you will need to either use an expo [development build](https://docs.expo.dev/development/introduction/) or a [prebuild](https://docs.expo.dev/workflow/prebuild/).
+
+Once your application is using a development build or prebuild you are ready to install the SDK:
+
+```bash
+expo install @beyondidentity/bi-sdk-react-native
+```
+
+Add the SDK [config plugin](https://docs.expo.dev/guides/config-plugins/) to the [plugins array](https://docs.expo.dev/versions/latest/config/app/#plugins) of your app.{json,config.js,config.ts}:
+
+```json
+{
+  "expo": {
+    "plugins": [["@beyondidentity/bi-sdk-react-native"]]
+  }
+}
+```
+
+The SDK requires certain minimun native versions. You can set these requirments either with another plugin, [expo-build-properties](https://docs.expo.dev/versions/latest/sdk/build-properties/), or by modifing project [static files](https://docs.expo.dev/guides/config-plugins/#static-modification).
+
+<RNExpoInstallation/>
+
+## Initialize the React Native SDK
+
+Once you've installed the SDK, initialize the SDK. The SDK will need to be initialized before you can call any of the Embedded functions. A good place to initalize this is where you register your root component. You may also add a listener to log native events with `Embedded.logEventEmitter` after initializing.
```

Comment:
i suggest a slight edit to condense the first two sentences here: Once you've installed the SDK, initialize it so that you can call the Embedded functions.
Also typo: 'initalize' should be 'initialize'